### ISSUE TITLE
Can't Install on MacOS 12 (Monterey)

### ISSUE THREAD

**Issue by TheDiscordian** [ipfs/go-ipfs#8347]
### Checklist

- [X] This is a bug report, not a question. Ask questions on [discuss.ipfs.io](https://discuss.ipfs.io).
- [X] I have searched on the [issue tracker](https://github.com/ipfs/go-ipfs/issues?q=is%3Aissue) for my bug.
- [X] I am running the latest [go-ipfs version](https://dist.ipfs.io/#go-ipfs) or have an issue updating.

### Installation method

GitHub releases page

### Version

```Text
v0.9.1 darwin arm64
```

### Description

On MacOS 12 `/usr/local/bin` doesn't exist it seems, so I can't run `install.sh`.

```
disco@smol go-ipfs % sudo ./install.sh
mv: rename ./ipfs to /usr/local/bin/ipfs: No such file or directory
mv: fastcopy: open() failed (to): /usr/bin/ipfs: Operation not permitted
We cannot install ./ipfs in one of the directories /usr/local/bin /usr/bin
It seems that we do not have the necessary write permissions.
Perhaps try running this script as a privileged user:

    sudo ./install.sh

disco@smol go-ipfs % ls /usr/local/bin
ls: /usr/local/bin: No such file or directory
```

P.S. The new Github submission form made writing this harder.

**Comment by lidel** [comment]
I'm bumping this in priority – @johnnymatthews  hit the same problem.
We need to fix this before go-ipfs 0.13 ships.

**Comment by ajnavarro** [comment]
I was not able to reproduce the problem on a virtual machine, so I rented a Scaleway native M1 computer: https://www.scaleway.com/en/hello-m1/ and I got the error.

Tentative fix here: #8934

After creating directories using the `install.sh` script, everything worked as expected.

**Comment by TheDiscordian** [comment]
@SgtPooki Sorry for late response, forgot I opened this issue. Does that homebrew run the install.sh script?

@ajnavarro #8934 lgtm, that's exactly the type of fix I was looking for.
